code-challenges.test.js

```diff
@@ -13,38 +13,180 @@
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
-// a) Create a test with an expect statement using the variable provided. 
-// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// a) Create a test with an expect statement using the variable provided. 
+// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
```

Comment:
The test structure looks good. however, this question is requesting an additional jest matcher on the expect statement. There is a link.
 One of the jest matcher that you can use is `expect.arrayContaining(array)`
 So from the link the structure you can mirror with the setup of this test.
`expect(['Alice', 'Bob', 'Eve']).toEqual(expect.arrayContaining(expected))`
HOWEVER
Our structure will need to include the function call with the test variable as its argument. Here is an example. 
`expect(colors3(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))`
So the jest matcher is allowing any output of the array to be accepted as long as the values are "yellow", "blue", "pink", "green". The values can be in any order but those are the only values that can be inside the array for the output to be correct.